src/core/mod.rs

```diff
@@ -423,3 +441,13 @@ impl Drop for BPF {
         };
     }
 }
+
+const SYSCALL_PREFIXES: [&str; 7] = [
```

Comment:
Move this up toward the top of the file - between the use declarations and the first struct declaration (following the whitespace on line 25) would be better